client/my-sites/earn/memberships/index.jsx

```diff
@@ -474,11 +474,11 @@ class MembershipsSection extends Component {
 			<div className="memberships__onboarding-wrapper">
 				<div className="memberships__onboarding-column-info">
 					<div className="memberships__onboarding-header">
-						{ this.props.translate( 'Introducing Recurring Payments.' ) }
+						{ this.props.translate( 'Introducing Payments.' ) }
 					</div>
 					<p className="memberships__onboarding-paragraph">
 						{ this.props.translate(
-							'Start collecting subscription payments! Recurring Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to collect paying subscribers.'
+							'Start selling and collecting subscription payments! Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to sell products or collect paying subscribers.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.